templates/admin-page.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Template for the Admin page.
+ *
+ * @package plugin-check
+ */
+
+if ( empty( $available_plugins ) ) {
+	return;
+}
```

Comment:
Let's remove this. Imagine a site has no plugins installed. It would be a bad experience showing the user a blank admin page.

Reply:
Done